stdlib/test/builtin/test_float_literal.mojo

```diff
@@ -95,6 +92,12 @@ def test_round():
     assert_equal(FloatLiteral.__round__(-1.198, 2), -1.2)
     assert_equal(FloatLiteral.__round__(-1.123, 2), -1.12)
 
+    # Test rounding to nearest even number
+    assert_equal(FloatLiteral.__round__(1.5, 0), 2.0)
+    assert_equal(FloatLiteral.__round__(2.5, 0), 2.0)
+    assert_equal(FloatLiteral.__round__(-2.5, 0), -2.0)
+    assert_equal(FloatLiteral.__round__(-1.5, 0), -2.0)
```

Comment:
Please add tests with negative value of `ndigits`.

Reply:
Actually, negative `ndigits` was not supported. Implemented here: https://github.com/msaelices/mojo/commit/1657b19a075259fc2908a65e8d539fba68fb8db4